cuphead/src/lib.rs

```diff
@@ -348,6 +359,22 @@ async fn tick<'a>(
         {
             reset();
         }
+
+        if scene == SCENE_SCOREBOARD {
+            monitor_star_skip(memory, measured_state);
+        }
+    }
+
+    if (settings.display_star_skip_counter_as_decimal) {
+        set_variable(
+            "star skip counter",
+            &format!("{}", measured_state.star_skip_counter_decimal as f32 / 6.0),
+        );
+    } else {
+        set_variable(
+            "star skip counter",
+            &format!("{}", measured_state.star_skip_counter),
+        );
     }
```

Comment:
slight improvement here
```suggestion
    let counter = if (settings.display_star_skip_counter_as_decimal) {
        &format!("{}", measured_state.star_skip_counter_decimal as f32 / 6.0)
    } else {
        &format!("{}", measured_state.star_skip_counter)
    };
    
    set_variable("star skip counter", counter);
```